RxSwift/Observables/ObserveOn.swift

```diff
@@ -101,7 +101,36 @@ final fileprivate class ObserveOnSink<O: ObserverType> : ObserverBase<O.E> {
             _scheduleDisposable.disposable = self._scheduler.scheduleRecursive((), action: self.run)
         }
     }
-    
+
+    #if swift(>=3.2)
+    func run(_ state: Void, recurse: () -> Void) {
```

Comment:
Yeah the 3.2 case is backwards compatible with 3.1 from what I've seen. I left the else case to have things 1:1 stay the same for pre–3.2 versions.